api/controllers/routes.go

```diff
@@ -35,6 +35,7 @@ func NewRouter() (router *mux.Router) {
 	router.HandleFunc("/apps/{app}/processes/{process}", api("process.stop", ProcessStop)).Methods("DELETE")
 	router.HandleFunc("/apps/{app}/processes/{process}/run", api("process.run.detach", ProcessRunDetached)).Methods("POST")
 	router.HandleFunc("/apps/{app}/releases", api("release.list", ReleaseList)).Methods("GET")
+	router.HandleFunc("/apps/{app}/releases", api("release.list", ReleaseList)).Methods("GET").Queries("limit", "{limit:[0-9]+}")
```

Comment:
I stuck this in a [separate commit](https://github.com/convox/rack/pull/1593/commits/7e9ae9a623668cb20adcc39981ea31164cff1d6d) because I wasn't sure if we'd want to restrict the param values with a regex, and I didn't find precedents to reference in convox/rack or convox/console.

```diff
-	router.HandleFunc("/apps/{app}/releases", api("release.list", ReleaseList)).Methods("GET").Queries("limit", "")
+	router.HandleFunc("/apps/{app}/releases", api("release.list", ReleaseList)).Methods("GET").Queries("limit", "{limit:[0-9]+}")
```